views/index.cr

```diff
@@ -1,12 +1,13 @@
 require "ecr/macros"
 
 class Views::Index
-  getter :crystal_repos, :repos_count, :sort
+  getter :crystal_repos, :repos_count, :sort, :filter
 
-  def initialize(repos_data, sort)
+  def initialize(repos_data, sort, filter = nil)
```

Comment:
In what situation do we want to initialize this without passing it a filter? Also, if we do, what about it being an empty string?